### ISSUE TITLE
Basic search result fields to ensure the largest possible returns

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#6020]
Suggest that a minimum number of fields is the default search option -- make a new button on the search page for ```Basic Search``` which will ensure that users dont accidentally limit their results by choosing too many optional fields. Also on the Customize Form a button option allowing users to reset to this Basic Search 

here's the suggested list of fields (should be kept to 7-8 MAX):
 - GUID
 - Identified As
 - Country
 - State/province
 - county
 - began date
 - parts

I know it's biocollection-centric but this does not affect anyone's customizations. In the future we could have a cultural Basic search option, too.
 let the arguing/discussions start!

**Comment by Jegelewicz** [comment]
To me, it is more locality centric. Taking up 3 spots with locality seems unfortunate? Do we not have some sort of summarized locality that might work for the results?

**Comment by mkoo** [comment]
>unfortunate
geography is never unfortunate! but fair enough, remove county then (not a core field)? concatenated hg+locality is not an option on the form

**Comment by dustymc** [comment]
Is this an addition to the existing "core"?

>new button

Is this in addition to...

<img width="96" alt="Screenshot 2023-03-17 at 8 00 55 AM" src="https://user-images.githubusercontent.com/5720791/225942153-945e745d-6079-4fd0-9fce-21a5292bd1ac.png">

> In the future we could have a cultural 

They are way ahead of the game and have two search profiles:

https://arctos.database.museum/search.cfm?sp=uam_eh
https://arctos.database.museum/search.cfm?sp=uam_art

I have been advertising that possibility since before the form came to be public, but IDK if anyone (other than the "culture people" obviously) is hearing me. Note that those options control search and results layouts, and can invoke the form with values. We can have one for people who are really into probably-incorrect asserted geography and one for those who aren't, and both for mammal collections, and both for THAT collection, and etc., etc. (Also previously mentioned, I'll need to do something different if we end up with more than ~a dozen profiles - and maybe this is dumb and we should just be pointing to the API????????)

https://github.com/ArctosDB/arctos/issues/5661 might be a more user-defined path to the same place.

>summarized locality

https://github.com/ArctosDB/arctos/issues/5460 - someone's got to ask for it, and we've got to find a time to dedicate the CPU to a global refresh. (Or maybe I should just update the things that are clear and deal with the cache-lag????)

**Comment by mkoo** [comment]
![Firefox_Screenshot_2023-03-17T15-59-27 077Z](https://user-images.githubusercontent.com/2523089/225956568-f8e1c35e-23b1-4266-a5d3-07dcdba63ff2.png)
I was originally talking about a new button at the top.

Could be the same basic blue button as Check All but Check Basic 

Also....

If that EH and ART already has that why is that hidden? Let's add an option to the top!

![Firefox_Screenshot_2023-03-17T16-02-10 328Z](https://user-images.githubusercontent.com/2523089/225957234-482c7bb6-f4cb-4bec-8987-17026b787e22.png)
Add  the ```Cultural Search option``` ```Art Search option```

I will file another for that actually (issue creep!) 

The latter two issues you reference-- that's the next conversation we need to have

**Comment by campmlc** [comment]
I would love to request a customized search form for mammal etc. That could solve the problem of knowing which customized fields to include?

**Comment by mkoo** [comment]
Actually the place to have these collection specific basic search and results would be on the respective Customize flyouts like at the top of this:
![Firefox_Screenshot_2023-03-17T16-53-39 826Z](https://user-images.githubusercontent.com/2523089/225968996-a917d548-513e-4f38-b96b-952f3e13006e.png)


I'm filing a new issue before Dusty starts going cross-eyed at me...

**Comment by dustymc** [comment]
>I would love to request

Get me enough data to fill out this..

```
create table core.cf_cat_rec_srch_profile (
    cf_cat_rec_srch_profile_id serial not null,
    profile_name varchar not null,
    creator varchar(255),
    description varchar(255),
    search_fields varchar,
    results_columns varchar
);
```
in the next hour or so, or use the new button to build your own after that.

Crazy idea: dump all this stuff, leave only the 'profiles' option (in test now, prod very soon). It's the same functionality as the two customize buttons, and users who aren't ready for that can still "customize" by following a link (that anyone can make).

**Comment by campmlc** [comment]
Could these saved profiles or settings be shared? So if MSB Mammals comes up with customized Search and Results settings/profile, the custom Search/Results profile could be shared, eg on our website, so that users could select it during a search to immediately search and download precustomized data fields?

**Comment by dustymc** [comment]
I think this is all available in production, tentatively closing.
